processor/src/main/java/com/linecorp/decaton/processor/ProcessorsBuilder.java

```diff
@@ -75,6 +81,21 @@ public ProcessorsBuilder(String topic, TaskExtractor<T> taskExtractor, TaskExtra
      * @return an instance of {@link ProcessorsBuilder}.
      */
     public static <T> ProcessorsBuilder<T> consuming(String topic, TaskExtractor<T> taskExtractor) {
+        return consuming(topic, new StringDeserializer(), taskExtractor);
+    }
+
+    /**
+     * Create new {@link ProcessorsBuilder} that consumes message from topic expecting tasks of type
+     * which can be parsed by valueParser.
+     * @param topic the name of topic to consume.
+     * @param keyDeserializer the deserializer for key that implements {@link Deserializer}.
```

Comment:
Let's mention that whatever the representation of key in the record is, it needs to be instantiated into a `String`.